FreeRTOS-Plus/Test/FreeRTOS-Plus-TCP/Integration/Full-TCP-Suite/main.c

```diff
@@ -46,11 +46,11 @@
 #include "FreeRTOS_IP.h"
 #include "FreeRTOS_Sockets.h"
 #include "FreeRTOS_DHCP.h"
-#include "demo_logging.h"
+#include "logging.h"
```

Comment:
The only real change here is `demo_logging.h` -> `logging.h`. The rest of the changes are from uncrustify.